.github/workflows/tests.yml

```diff
@@ -0,0 +1,64 @@
+name: PHPUnit
```

Comment:
I would name it `CI` rather than `PHPUnit`, as we might add more jobs doing something else than running PHPUnit tests.